build/src/components/05-gallery/gallery.js

```diff
@@ -1,68 +1,182 @@
-import 'swiper/dist/js/swiper.js';
+import Swiper from 'swiper/dist/js/swiper.js';
 import 'featherlight/release/featherlight.min';
-import enquire from 'enquire.js';
 import log from 'loglevel';
 import { debounce } from 'underscore';
 import GalleryPI from './gallery-pi';
-import { SwiperGalleryBreakpoints } from '../../00-base/02-breakpoints/breakpoints'
-
-let breakpoints = new SwiperGalleryBreakpoints();
-
-// Limit smaller than 200 can trigger flickering behavior when resizing the
-// gallery.
-let debounce_limit = 200;
+import GalleryAds from './gallery-ads';
 
 /**
- * An instance of a gallery.
+ * Default gallery.
  */
 class Gallery {
+
+  /**
+   * Gallery type.
+   *
+   * @returns {string}
+   */
   static get type() {
     return 'default';
   }
 
   /**
-   * Attaches the class to the elements of the given selector.
+   * Mobile breakpoint.
+   *
+   * @returns {number}
+   */
+  static get mobileBreakpoint() {
+    return 534;
+  }
+
+  /**
+   * Wait parameter for debounce.
```

Comment:
Add comment: Limit smaller than 200 can trigger flickering behavior when resizing the gallery